src/js/utils/colors.js

```diff
@@ -1,14 +1,20 @@
-export const normalizeColor = (color, theme) => {
+export const normalizeColor = (color, theme, forceDark) => {
   const colorSpec =
     theme.global && theme.global.colors[color] !== undefined
       ? theme.global.colors[color]
       : color;
   // If the color has a light or dark object, use that
   let result = colorSpec;
   if (colorSpec) {
-    if (theme.dark && colorSpec.dark !== undefined) {
+    if (
+      (forceDark === true || (forceDark === undefined && theme.dark)) &&
```

Comment:
I would think that `(forceDark === false && theme.dark)` should also be accepted in this condition; If I'm not forcing the dark mode but the theme itself is already dark, then the result will be the dark color.

Reply:
If you're not forcing the dark mode, then forceDark will be undefined. If it is explicitly set to `false`, then it takes precedence.

Reply:
Maybe the naming is confusing to me then, I think that if `forceDark === false` it means I'm not forcing it to happen, but if it is happening by itself (i.e. theme.dark) then that is ok. Would it make sense to rename `forceDark` to `isDark` or `setDark`?

Reply:
Hmm... I was thinking "forceDark" was even stronger, that you were saying you wanted "dark" to be forced to whatever "forceDark" indicated. Maybe making the argument to be just "dark" would be clearer?

Reply:
Yes, it will make more sense to me if it is just `dark`. Once `dark` is set to false then it will make sense not to pick the dark theme.

Reply:
OK. Renamed, and added a comment explaining more.